gyp/LICENSE

```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2019 Ujjwal Sharma. All rights reserved.
```

Comment:
Question: Should we add Node.js Contributors?  I am happy either way.

Reply:
Yeah, that's a bit weird, and it's in nodejs/gyp-next too. Can we get that fixed so it doesn't look like a personal project?